R/.importConservation.R

```diff
@@ -0,0 +1,55 @@
+.importConservation <- function(fName,
```

Comment:
What am I looking at here? There are two files: ```.importConservation.R``` and ```importConservation.R``` both implementing ```.importConservation()```. Why?

I'll only be commenting on the other file, below.